frame/elections-phragmen/src/lib.rs

```diff
@@ -258,46 +300,72 @@ decl_module! {
 		const TermDuration: T::BlockNumber = T::TermDuration::get();
 		const ModuleId: LockIdentifier  = T::ModuleId::get();
 
-		/// Vote for a set of candidates for the upcoming round of election.
+		/// Vote for a set of candidates for the upcoming round of election. This can be called to
+		/// set the initial votes, with `would_create = true`, or update already existing votes
+		/// with `would_create = false`.
+		///
+		/// Upon initial voting, `value` units of `who`'s balance is locked and a bond amount is
+		/// reserved.
 		///
 		/// The `votes` should:
 		///   - not be empty.
-		///   - be less than the number of candidates.
+		///   - be less than the number of possible candidates. Note that all current members and
+		///     runners-up are also automatically candidates for the next round.
 		///
-		/// Upon voting, `value` units of `who`'s balance is locked and a bond amount is reserved.
 		/// It is the responsibility of the caller to not place all of their balance into the lock
 		/// and keep some for further transactions.
 		///
 		/// # <weight>
-		/// #### State
-		/// Reads: O(1)
-		/// Writes: O(V) given `V` votes. V is bounded by 16.
+		/// Base weight: 50us
+		/// State reads:
+		/// 	- Candidates.len() + Members.len() + RunnersUp.len()
+		/// 	- Voting (is_voter)
```

Comment:
So as discussed, I assume that reading just the length of some storage is different than reading it later. 

Correct me if I am wrong.